.github/workflows/test_suite_linux_develop.yml

```diff
@@ -0,0 +1,157 @@
+name: Test Suite Linux
+
+on:
+  schedule:
+    - cron:  '0 23 * * *'
+
+defaults:
+  run:
+    shell: bash
+jobs:
+  cleanup-runs:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: rokroskar/workflow-run-cleanup-action@master
+      env:
+        GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+  build:
+    needs: cleanup-runs
+    if: ${{ github.event_name == 'push' || github.event_name == 'schedule' || contains(github.event.pull_request.labels.*.name, 'test distribution') || contains(github.event.pull_request.labels.*.name, 'test suite') || contains(github.event.pull_request.labels.*.name, 'test ros') || contains(github.event.pull_request.labels.*.name, 'test world update') }}
```

Comment:
Some parts such as this `if` could be removed/simplified on the develop workflow, but I prefer to keep them to keep the difference between workflows as small as possible.